www/data_module/src/specification.constant.coffee

```diff
@@ -1,272 +1,280 @@
+                    'tags'
+                ]
+                root: true
+                paths: [
+                    'forceschedulers'
+                    'buildrequests'
+                    'masters'
+                    'masters/n:masterid'
+                    'buildslaves'
+                    'buildslaves/i:name'
+                    'buildslaves/n:buildslaveid'
+                    'builds'
+                    'builds/n:number'
+                    'builds/n:number/steps'
+                    'builds/n:number/steps/i:name'
+                    'builds/n:number/steps/i:name/logs'
+                    'builds/n:number/steps/i:name/logs/i:slug'
+                    'builds/n:number/steps/i:name/logs/i:slug/contents'
+                    'builds/n:number/steps/i:name/logs/i:slug/raw'
+                    'builds/n:number/steps/n:number'
+                    'builds/n:number/steps/n:number/logs'
+                    'builds/n:number/steps/n:number/logs/i:slug'
+                    'builds/n:number/steps/n:number/logs/i:slug/contents'
+                    'builds/n:number/steps/n:number/logs/i:slug/raw'
+                ]
+                static: true
+            buildrequests:
+                id: 'buildrequestid'
+                fields: [
+                    'buildrequestid'
+                    'builderid'
+                    'buildsetid'
+                    'claimed'
+                    'claimed_at'
+                    'claimed_by_masterid'
+                    'complete'
+                    'complete_at'
+                    'priority'
+                    'results'
+                    'submitted_at'
+                    'waited_for'
+                ]
+                root: true
+                paths: [
+                    'builds'
+                ]
+                static:
+                    complete: true
+            buildsets:
+                id: 'bsid'
+                fields: [
+                    'bsid'
+                    'complete'
+                    'complete_at'
+                    'external_idstring'
+                    'parent_buildid'
+                    'parent_relationship'
+                    'reason'
+                    'results'
+                    'sourcestamps'
+                    'submitted_at'
+                ]
+                root: true
+                paths: [
+                    'properties'
+                ]
+                static:
+                    complete: true
+            buildslaves:
+                id: 'buildslaveid'
+                fields: [
+                    'buildslaveid'
+                    'configured_on'
+                    'connected_to'
+                    'name'
+                    'slaveinfo'
+                ]
+                root: true
+                paths: []
+                static: true
+            changes:
+                id: 'changeid'
+                fields: [
+                    'changeid'
+                    'author'
```

Comment:
I use these fields to create the IndexedDB tables, it's possible to put an object with additional fields beside these, but those won't be query-able. In our cases it doesn't really matters, because I filter the data with a similar algorithm to the server side filtering, so it's not working with the IndexedDB API directly but on all the data in a table. I wan't to use these to create mocked data, that we can use to help the offline development, without a proxy.